modules/etcd-defrag.adoc

```diff
@@ -63,6 +71,11 @@ A Prometheus alert indicates when you need to use manual defragmentation. The al
 Defragmenting etcd is a blocking action. The etcd member will not respond until defragmentation is complete. For this reason, wait at least one minute between defragmentation actions on each of the pods to allow the cluster to recover.
 ====
 
+Before you proceed with defragmentation, you can also check the etcd DB size in MiB that would be freed/released if defragmentation were to be performed with the next PromQL expression:
```

Comment:
I'd add this information above the warning and revise it to comply with style guidelines. For example, the revision would look like this:

A Prometheus alert indicates when you need to use manual defragmentation. The alert is displayed in two cases:

* When etcd uses more than 50% of its available space for more than 10 minutes

* When etcd is actively using less than 50% of its total database size for more than 10 minutes

You can also determine whether defragmentation is needed by checking the etcd database size in MB that will be freed by defragmentation: `(etcd_mvcc_db_total_size_in_bytes - etcd_mvcc_db_total_size_in_use_in_bytes)/1024/1024`